data/json/items/tool/cooking.json

```diff
@@ -1067,6 +1067,22 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "water_filter_sand",
+    "type": "TOOL",
+    "name": { "str": "slow-sand water filter" },
+    "//": "Slow sand biofilter: https://wwwnc.cdc.gov/travel/yellowbook/2024/preparing/water-disinfection#:~:text=Figure%202%2D02.%20Emergency%20gravel%20and%20sand%20filter",
+    "description": "An improvised water filter made from a plastic bucket filled with a layer of sand and a layer of gravel.  It can be used to remove particulates and some pathogens from water, but it is not 100% effective.",
+    "//2": "Density of sand is around 1.6g/cm^3. Gravel is similar. pi * 5in * 5in * (10 in + 4 in) * 1.6 g/cm^3 ~= 29 kg. Add 1 kg for the bucket itself.",
+    "weight": "14800 g",
```

Comment:
This comment wasn't updated. It doesn't equal 14800 g but 29 kg.

Reply:
See #74323, the comment is right but the weight of this item is wrong, so I've fixed it.

Reply:
Are you sure? It was changed here before https://github.com/CleverRaven/Cataclysm-DDA/pull/72252#discussion_r1586313819 to fix density apparently.

Reply:
> See #74323, the comment is right but the weight of this item is wrong, so I've fixed it.

```
    "components": [
      [ [ "bucket_5gal", 1 ] ],
      [ [ "material_sand", 1250 ] ],
      [ [ "material_gravel", 500 ] ],
      [ [ "cotton_patchwork", 4 ], [ "wire_mesh", 4 ] ]
    ]
```
In #74323 , you only changed the weight of the filter, not its recipe, right? Then you are making a 29800 g item with these components which only weighs 14835 g in total🤔

Reply:
Thanks for noticing, #74332 should address it.